### PR TITLE
Simplify predictions / scoring code

### PR DESCRIPTION
**Pull request by JedGrabman**
Uses the new version of `get_covidhub_predictions` by querying multiple forecasters at once.
Also makes use of `evaluate_covid_predictions`, by evaluating multiple signals at once.

`covidcast` also now returns actual data for territories, so `CovidHubUtils` only needs to be used for national level data.

Deletes `predictions.R`
Refactor `score.R`

### CODE REVIEW

File: Report/create_reports.R [hunk 1]
```diff
@@ -1,37 +1,100 @@
...
     TRUE~prediction_cards_filename
 )
 
-source("predictions.R")
-create_prediction_cards(prediction_cards_filepath)
+forecasters = c(get_covidhub_forecaster_names(designations = "primary"),
+                "COVIDhub-baseline")
+locations = covidHubUtils::hub_locations
+
+# also includes "us", which is national level data
```
Comment (kateharwood) [inline Report/create_reports.R:30]: Is this removing "US" though?

Comment (kateharwood) [reply Report/create_reports.R:30]: Nevermind, I see we are filtering that out elsewhere.

**Comment by kateharwood** [comment]
This does not adjust the memory usage as far as you know, right? Maybe we should test this with the deploy workflow to make sure everything still runs through?

**Comment by JedGrabman** [comment]
As far as I know, there shouldn't be a memory impact, but I kicked off the deploy workflow to test it: https://github.com/cmu-delphi/forecast-eval/runs/2222184360?check_suite_focus=true

**Review by kateharwood — APPROVED** [review]

**Pull request merged**